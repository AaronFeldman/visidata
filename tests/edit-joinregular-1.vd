sheet	col	row	longname	input	keystrokes	comment
			open-file	tests/data1.tsv	o	open file
			open-file	tests/data2.tsv	o	open file
data2			sheets-stack		S	open Sheets Sheet
sheets		キdata2	select-row		s	select current row
sheets		キdata1	select-row		s	select current row
sheets			columns-selected		gC	open Columns Sheet with all columns from selected sheets
all_columns		キdata2,Key	key-selected		g!	toggle selected rows as key columns on source sheet
all_columns		キdata1,Key	key-selected		g!	toggle selected rows as key columns on source sheet
all_columns			quit-sheet		q	quit current sheet
sheets			join-sheets	full	&	merge the selected sheets with visible columns from all, keeping rows according to jointype
data2+data1	A		key-col		!	toggle current column as a key column
data2+data1			sort-keys-asc		g[	sort ascending by all key columns
data2+data1	A		key-col		!	toggle current column as a key column
data2+data1	C	キ2	edit-cell	a3	e	
