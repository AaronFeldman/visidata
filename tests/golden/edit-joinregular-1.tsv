Key	data2_C	data2_D	data1_A	data1_B
1			a1	b1
2	a3	b2	c1	d1
2	a3	b2	e1	f1
3	c2	d2		
